tooling/linux_repro_build_compare.sh

```diff
@@ -94,7 +94,9 @@ downloadTooling() {
 }
 
 checkAllVariablesSet() {
-    [ -z "$SBOM" ] || [ -z "${BOOTJDK_VERSION}" ] || [ -z "${TEMURIN_BUILD_SHA}" ] || [ -z "${TEMURIN_BUILD_ARGS}" ] || [ -z "${TEMURIN_VERSION}" ] && echo "Could not determine one of the variables - run with sh -x to diagnose" && sleep 10 && exit 1
+  if [ -z "$SBOM" ] || [ -z "${BOOTJDK_VERSION}" ] || [ -z "${TEMURIN_BUILD_SHA}" ] || [ -z "${TEMURIN_BUILD_ARGS}" ] || [ -z "${TEMURIN_VERSION}" ]; then
+      echo "Could not determine one of the variables - run with sh -x to diagnose" && sleep 10 && exit 1
```

Comment:
why need a "sleep 10" if it eventually does "exit 1"

Reply:
I think was just originally so things didn't scroll off the screen if you were watching it...